Test/WebTestCase.php

```diff
@@ -342,6 +370,11 @@ private function loadFixtureClass(Loader $loader, string $className)
             return;
         }
         $loader->addFixture($fixture);
+        if ($fixture instanceof DependentFixtureInterface) {
```

Comment:
This was removed in 65ec0c4c847640779549edc88043f06ad74d3211
Why adding it again?